packages/react-chat/src/components/SystemResponse/styled.ts

```diff
@@ -95,4 +95,24 @@ export const List = styled(tag('div', 'list'), {
       marginBottom: 0,
     },
   },
+
+  variants: {
+    mode: {
+      bubble: {
+        [`& ${Message.Container}`]: {
+          maxWidth: 248,
+        },
+      },
+      embedded: {
+        // minus avatar and paddings, minus timestamp width
+        maxWidth: 'calc(100% - 54px - 50px)',
+        [`& ${Message.Container}`]: {
+          maxWidth: '100%',
+        },
```

Comment:
I think there's a better way of doing this where you just make it `flex: 1`. Try and see if that works and doesn't break anything.
It's better to use dynamic values